server/db/models/user.js

```diff
@@ -16,6 +20,10 @@ const User = db.define('user', {
       return () => this.getDataValue('password')
     }
   },
+  profilePic: {
+    type: Sequelize.STRING,
+    defaultValue: 'https://bit.ly/37Lab4d'
+  },
```

Comment:
change properties to example ---> 

`email: 'dave@email.com',
    firstName: 'Dave',
    lastName: 'Chappelle',
    password: '12345',
    imgUrl:
      'https://www.indiewire.com/wp-content/uploads/2019/08/Screen-Shot-2019-08-26-at-4.08.20-PM.png',
    isAdmin: false`